app/src/main/java/com/hedvig/app/util/featureflags/TestFeatureFlagProvider.kt

```diff
@@ -0,0 +1,16 @@
+package com.hedvig.app.util.featureflags
+
+object TestFeatureFlagProvider : FeatureFlagProvider {
```

Comment:
Is this still needed when we're using DI? I guess we can probably just mock on the DI-level:

```kotlin
val mock = mocck<FeatureRuntimeBehavior>()
every { mock.isFeatureEnabled(Feature.MOVING_FLOW) } returns true
```

Reply:
Right, that makes more sense.